docs/user/index.rst

```diff
@@ -4,6 +4,10 @@ Read the Docs: Documentation Simplified
 .. meta::
    :description lang=en: Automate building, versioning, and hosting of your technical documentation continuously on Read the Docs.
 
+.. raw:: html
+
+   <a style="display: none;" rel="me" href="https://fosstodon.org/@readthedocs">Mastodon</a>
+
```

Comment:
Would it be possible to put this further down - in this block?

```
* **Getting involved with Read the Docs**:
  :doc:`/glossary` |
  :doc:`/gsoc`|
  :doc:`Developer Documentation <rtd-dev:index>`
```

Reply:
Why do you think it's better to put it there instead? The link is hidden and not shown to the user anyways. Note the `style="display: none"`. We don't have social link on that page.